app/upgrades_test.go

```diff
@@ -0,0 +1,41 @@
+package app_test
+
+import (
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/kava-labs/kava/app"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpgradeCommunityParams_Mainnet(t *testing.T) {
+	require.Equal(
+		t,
+		sdkmath.LegacyZeroDec().String(),
+		app.CommunityParams_Mainnet.StakingRewardsPerSecond.String(),
+	)
+
+	require.Equal(
+		t,
+		// Manually confirmed
+		"317097.919837645865043125",
```

Comment:
Also, verified this value - ty for the unit test here to verify the above math 🚀